astroplan/core.py

```diff
@@ -22,109 +38,152 @@
 
 class Observer(object):
     """
-    Some comments.
+    A container for information about an observatory's location and environment.
```

Comment:
also, maybe "observer" instead of "observatory"?  After all, it could just be a person outside with their eyes...